feature-assets/src/main/java/io/novafoundation/nova/feature_assets/di/AssetsFeatureModule.kt

```diff
@@ -141,4 +156,20 @@ class AssetsFeatureModule {
         coinPriceRepository = coinPriceRepository,
         poolAccountDerivation = poolAccountDerivation
     )
+
+    @Provides
+    fun provideInitialSwapFlowExecutor(
```

Comment:
Should this `provideSwapExecutor` and  be `@FeatureScope`?

Reply:
Yes, of course. Thank you